abcclassroom/clone.py

```diff
@@ -12,11 +12,11 @@
 from . import github as gh
 
 
-def clone_or_update_repo(organization, repo, clone_dir, skip_existing):
+def clone_or_update_repo(organization, repo, clone_dir, update_existing):
```

Comment:
thank you @kcranston !! so when i tried to run this in the notebook it was telling me i had to specify a value for update_existing (skip_existing). We don't have a default value for it as it stands. i can play with this some more but i was thinking we should have to specify update_existing and have some default value which you can overwrite if you wish? 

this all looks great however but that was the issue that i had.

Reply:
You don't happen to have the exact error, do you?

Reply:
I just tested it -  i updated the hub last night to 0.1.9 and the error is now gone!!

i have a notebook  on the hub that tests the notebook version of using abc-classroom!! so you can ignore my comment.